wsl-vpnkit

```diff
@@ -98,6 +99,9 @@ vpnkit () {
     if [ "$VPNKIT_DEBUG" ]; then
         CMD="$CMD"' --debug'
     fi
+    if [ "$HTTP_PROXY_CONFIG_PATH" ] && [ "$(file -bLkr --mime-type "$HTTP_PROXY_CONFIG_PATH")" = 'application/json' ]; then
```

Comment:
I don't think checking the file mime type is worth adding a dependency to `file`. I think just checking the existence of the file is ok.

Reply:
✔